### DISCUSSION TITLE
Interoperability with Frictionless Standards

### DISCUSSION THREAD

**Discussion by mnauge** [TheELNConsortium/TheELNFileFormat#50] [Ideas]
Le format proposé est-il compatible avec les standards et outils préconisés par Frictionless ?

https://frictionlessdata.io/standards/#standards-toolkit

**Comment by NicolasCARPi** [comment]
Hello (please keep it in english),

> Is the proposed format compatible with Frictionless standards?

I believe the use of JSON-LD in RO-Crate is even better than what frictionless is proposing. The RO-Crate whitepaper mentions Frictionless and its shortcomings: 

> We have identified a gap for a solid base format for data packaging that also allow communities to
build domain-specific solutions. Frictionless data [8] could arguably fill this gap, with mature
specifications and a strong design philosophy, however as an independent JSON format it does not
fully apply Linked Data principles, and would be harder to use in FAIR integrations and extensions.

(from https://zenodo.org/record/3337883)

Also:


> The main addition in the upcoming 1.2 release of the RO-Crate specifications will be the formalization of [profiles](https://www.researchobject.org/ro-crate/1.2-DRAFT/profiles) for different categories of crates. Additional entity types have been requested by users, e.g. workflow runs, business workflows, containers and software packages, tabular data structures; these are not always matched well with existing Schema.org types but may benefit from other vocabularies or even separate metadata files, e.g. from [Frictionless Data](https://frictionlessdata.io/). We will be further aligning and collaborating with related research artefact description efforts like [CodeMeta](https://codemeta.github.io/) for software metadata, [Science-on-Schema.org](https://science-on-schema.org/) [[125](https://www.researchobject.org/2021-packaging-research-artefacts-with-ro-crate/manuscript.html#ref-PWcjZsZl)] for datasets, [FAIR Digital Objects](https://fairdo.org/) [[75](https://www.researchobject.org/2021-packaging-research-artefacts-with-ro-crate/manuscript.html#ref-k5pGkxd5)] and activities in [EOSC task forces](https://www.eosc.eu/task-force-faq) including the EOSC Interoperability Framework [[16](https://www.researchobject.org/2021-packaging-research-artefacts-with-ro-crate/manuscript.html#ref-aCye3KpE)].

(from https://www.researchobject.org/2021-packaging-research-artefacts-with-ro-crate/manuscript.html)
